ios/fastlane/metadata/en-US/release_notes.txt

```diff
@@ -1 +1,2 @@
 - Added blurred preview to messages with single image attachment.
+- Fixed selected application language not being persisted.
```

Comment:
0.1.3 versions weren't released, thus should list 0.1.3 plus 0.1.4 changes there.